.gitignore

```diff
@@ -5,3 +5,4 @@ db.sqlite3
 .DS_Store
 /uploads
 .coverage
+/virtualenvironments
```

Comment:
Probably it would be better to keep your virtual environment in some other folder? Because it depend on person to person.